### PR TITLE
Export archived orders workflow

### PR DESCRIPTION
**Pull request by EvheniiaKolkhiieva**
## Purpose of this pull request

This pull request (PR) describes in what statuses orders can be archived, how to export archived orders, and fix the wrong button title for the "Restore an archived order" block
## Magento release version

- [x] 2.4.x

   Specify a patch release number, if applicable:

- [ ] 2.3.x

   Specify a patch release number, if applicable:

## Product editions

Is this update specific to a product edition?

- [x] Magento Open Source only
- [x] Magento Commerce only

Is this update specific to an installed feature extension

- [x] B2B extension
- [x] Other feature set, please specify:

## Affected documentation pages

- https://docs.magento.com/user-guide/sales/order-archive.html

### CODE REVIEW

File: src/sales/order-archive.md [hunk 1]
```diff
@@ -52,6 +52,9 @@ _[Move Order to Archive]({% link configuration/sales/sales.md %})_
     ![]({% link images/images-ee/order-select-in-grid.png %}){: .zoom}
     _Mark Checkbox_
 
+{:.bs-callout-tip}
+Only orders with **Complete** or **Closed** status can be moved to the archive. Orders with other statuses will remain in the Order Management grid.
```
Comment (dmytro-ch) [inline src/sales/order-archive.md:56]: Technically an order with any other status can be moved to the archive. I would say it rather depends on the `Order Statuses to be Archived` system config.

**Review by dmytro-ch — CHANGES_REQUESTED** [review]

File: src/sales/order-archive.md (hunk 1 above)
Comment (EvheniiaKolkhiieva) [reply src/sales/order-archive.md]: Hi @dmytro-ch, thank you for the great tip.

I have modified this tip, according to your recommendation, could you please check and approve?

Comment (dmytro-ch) [reply src/sales/order-archive.md]: Hello @EvheniiaKolkhiieva,

Awesome, thank you!

**Review by dmytro-ch — APPROVED** [review]

**Review by dobooth — APPROVED** [review]

**Comment by dobooth** [comment]
@magento import code to magento-devdocs/merchdocs

File: src/sales/order-archive.md [hunk 2]
```diff
@@ -83,8 +86,20 @@ _[Move Order to Archive]({% link configuration/sales/sales.md %})_
     ![]({% link images/images-ee/order-archived-select-to-restore.png %}){: .zoom}
     _Select Order to be Restored_
 
-1. Set the **Actions** control value to `Remove to Archive`.
+1. Set the **Actions** control value to **Move to Order Management**.
+
+ Look for the message that the archived order has been removed from the archive.
+
+## To export archived order:
```
Comment (leslietilling) [inline src/sales/order-archive.md:93]:
```suggestion
## Export archived order
```

File: src/sales/order-archive.md [hunk 3]
```diff
@@ -83,8 +86,20 @@ _[Move Order to Archive]({% link configuration/sales/sales.md %})_
...
+
+1. On the _Admin_ sidebar, click **Sales**.
+
+1. In the _Operations_ section, choose **Orders**.
```
Comment (leslietilling) [inline src/sales/order-archive.md:97]:
```suggestion
1. On the _Admin_ sidebar, go to **Sales** > _Operations > **Orders**.
```

File: src/sales/order-archive.md [hunk 4]
```diff
@@ -83,8 +86,20 @@ _[Move Order to Archive]({% link configuration/sales/sales.md %})_
...
+
+1. In the button bar, click **Go to Archive**.
+
+1. In the action menu select **Export** drop-down.
 
-    Look for the message that the archived order has been removed from the archive.
+1. Select the desired format.
```
Comment (leslietilling) [inline src/sales/order-archive.md:103]: Replace with a single step:

1. In the action menu, click **Export**  and select the desired format.

**Comment by EvheniiaKolkhiieva** [comment]
@leslietilling, all edits have been implemented, you can check it.

Thank you!

File: src/sales/order-archive.md [hunk 5]
```diff
@@ -70,11 +73,9 @@ _[Move Order to Archive]({% link configuration/sales/sales.md %})_
     ![]({% link images/images-ee/order-archived-view.png %}){: .zoom}
     _View Archived Order_
 
-## To restore an archived order:
-
-1. On the _Admin_ sidebar, click **Sales**.
+## Restore an archived order:
 
-1. In the _Operations_ section, choose **Orders**.
+1. On the _Admin_ sidebar, go to **Sales** > _Operations > **Orders**.
```
Comment (leslietilling) [inline src/sales/order-archive.md:78]: My error on this one

```suggestion
1. On the _Admin_ sidebar, go to **Sales** > _Operations_ > **Orders**.
```

File: src/sales/order-archive.md [hunk 6]
```diff
@@ -83,8 +84,14 @@ _[Move Order to Archive]({% link configuration/sales/sales.md %})_
...
+1. Set the **Actions** control value to **Move to Order Management**.
+
+ Look for the message that the archived order has been removed from the archive.
+
+## Export archived order
+
+1. On the _Admin_ sidebar, go to **Sales** > _Operations > **Orders**.
 
-    Look for the message that the archived order has been removed from the archive.
+1. In the action menu, click Export and select the desired format.
```
Comment (leslietilling) [inline src/sales/order-archive.md:95]:
```suggestion
1. In the action menu, click **Export** and select the desired format.
```

File: src/sales/order-archive.md [hunk 7]
```diff
@@ -83,8 +84,14 @@ _[Move Order to Archive]({% link configuration/sales/sales.md %})_
...
 
-1. In the button bar, click **Return to Order Management**.
+1. Click **Export**.
```
Comment (leslietilling) [inline src/sales/order-archive.md:97]:
```suggestion
```

**Comment by magento-engcom-team** [comment]
@dobooth the branch with code successfully imported into`magento-devdocs/merchdocs` repository. Branch name: `imported-magento-merchdocs-1194`.

**Pull request merged**